python/libkvikio/pyproject.toml

```diff
@@ -55,3 +56,6 @@ requires = [
     "cmake>=3.26.4,!=3.30.0",
     "ninja",
 ] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project.entry-points."cmake.prefix"]
+libkvikio = "libkvikio"
```

Comment:
I didn't realize `scikit-build` allowed you to do this.

https://github.com/scikit-build/scikit-build-core/blob/8cbd92f430c20ad5e0d9725198a1397ab599437d/docs/cmakelists.md?plain=1#L83-L87

https://github.com/scikit-build/scikit-build-core/blob/8cbd92f430c20ad5e0d9725198a1397ab599437d/src/scikit_build_core/builder/builder.py#L129-L131

Interesting! If you know the history for why this re-uses the `project.entry-points` table (which I'd always thought was intended to provide CLI commands that call into Python code), instead of having `scikit-build-core`-specific configuration that just accepts a list of paths to append to `CMAKE_PREFIX_PATH`, I'd love to hear it.

Reply:
CLI commands are just one use case for entry points. In reality they can be used for many different things.

The reason a `scikit-build-core`-specific configuration isn't used is because `CMAKE_PREFIX_PATH` isn't used when buiilding *this* package, but rather when building packages that consume it. Some sort of metadata has to be installed along with this package. Entry points are the way to do that.

Reply:
ooooo I see. Thanks for explaining that!